index.php

```diff
@@ -10,9 +10,11 @@
 
 $errorMessage = "";
 if(count($_POST)>0) {
-    if (checkFormSubmission($_POST)[0]) {
+    $formSubmissionCheck = checkFormSubmission($_POST);
+    $dropdownSubmissionCheck = checkDropdownSubmission($_POST);
+    if ($formSubmissionCheck['result'] && $dropdownSubmissionCheck['result']) {
         addBottle($PDO, $_POST['item-name'], $_POST['purchase-location'], $_POST['type'], $_POST['purchase-date']);
-    } else $errorMessage = checkFormSubmission($_POST)[1];
+    } else $errorMessage = $formSubmissionCheck['message'] . $dropdownSubmissionCheck['message'];
```

Comment:
put a space or a new line between them so they're not able to look like this
![image](https://user-images.githubusercontent.com/25844355/153381831-79c6587a-4701-4d43-adfc-11dba63c710a.png)